web/src/helpers/fetch.ts

```diff
@@ -1,13 +1,32 @@
+import { getCsrfToken } from 'web/src/helpers/getCsrfToken';
+
 export class FetchError extends Error {
   constructor(public messages: string[], public code: number, public payload: Record<string, any>) {
     super(messages[0]);
     this.name = 'FetcherError';
   }
 }
 
+function fillCsrf(init?: RequestInit) {
```

Comment:
```suggestion
function createRequestWithCSRF(init?: RequestInit) {
```
Я думаю примерно такое имя должно быть